actionview/lib/action_view/helpers/tags/check_box.rb

```diff
@@ -38,6 +28,23 @@ def render
           end
         end
```

Comment:
Templating engines (like ERB's `<%= %>` interpolation syntax) will end up invoking `#to_s`. 

Alongside that expectation, this change poses an opportunity to define [`#render_in(view_context)`](https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/RenderingHelper.html#method-i-render-label-Rendering+Mode) to integrate with the newer Action View's `:renderable` Object rendering.

```ruby
def to_s
  render_in(@template_object)
end

def render_in(view_context)
  options = options_with_hidden_attribute

  include_hidden = options.delete("include_hidden") { true }
  checkbox = view_context.tag("input", options)

  if include_hidden
    hidden = hidden_field_for_checkbox(options)
    hidden + checkbox
  else
    checkbox
  end
end
```

With that change, form builder tag objects could be passed directly to `render` calls as position arguments or under the `:renderable` key.

Reply:
I'll make them compatible with `render_in` as well! Thanks for pointing that out.